tests/suites/test_suite_ssl.function

```diff
@@ -4062,6 +4062,8 @@ MBEDTLS_SSL_TLS1_3_LABEL_LIST
     TEST_ASSERT( (size_t) desired_length <= sizeof(dst) );
     TEST_ASSERT( (size_t) desired_length == expected->len );
 
+    USE_PSA_INIT( );
```

Comment:
This should be `PSA_INIT( );` - the function already depends on `MBEDTLS_SSL_PROTO_TLS1_3` so we know PSA is available and used.